Model.java

```diff
@@ -66,13 +538,222 @@ public void writeScheduleToFile(String fileName) throws IOException {
             FileWriter writer = new FileWriter(new File(fileName + ".json"));
             writer.write(sb.toString());
             writer.close();
+            System.out.println("------------------------------");
             System.out.println("Successfully wrote schedule to " + fileName + ".json");
         } catch (IOException e) {
             e.printStackTrace();
         }
     }
 
-    public void readScheduleToFile(String fileName) throws IOException {
```

Comment:
Maybe a FileIO class would be useful as well here, because the model class is more complicated with reading/writing to a file